ci/scripts/prepare-cached-bitstream.sh

```diff
@@ -19,6 +19,9 @@ BITSTREAM=HEAD ci/bazelisk.sh build --define bitstream=gcp_splice \
   //hw/bitstream:rom_mmi \
   //hw/bitstream:otp_mmi
 
+# Move the bitstream spliced with locally built ROM to the location expected by subsequent steps.
```

Comment:
Why not just copy it from its correct location and skip this step?

Reply:
sure we can clean this up if this fix works. for now I just wanted to see if my understanding was correct if this change (bold below) indeed fixes the issue:
* take `a` from the cache
* splice it with the freshly built rom to produce `b`
* **move `b` to `a`**
* copy `a` to some other place 
* all other targets use `a`